userspace/libsinsp/sinsp.h

```diff
@@ -218,7 +218,7 @@ class SINSP_PUBLIC sinsp : public capture_stats_source
 
 
 	/* Wrappers to open a specific engine. */
-	virtual void open_kmod(unsigned long driver_buffer_bytes_dim = DEFAULT_DRIVER_BUFFER_BYTES_DIM, const std::unordered_set<uint32_t> &ppm_sc_of_interest = {}, const std::unordered_set<uint32_t> &tp_of_interest = {});
+	virtual void open_kmod(const std::string& kmod_name = "", unsigned long driver_buffer_bytes_dim = DEFAULT_DRIVER_BUFFER_BYTES_DIM, const std::unordered_set<uint32_t> &ppm_sc_of_interest = {}, const std::unordered_set<uint32_t> &tp_of_interest = {});
```

Comment:
`SCAP_KERNEL_MODULE_NAME` is not visible here :/ perhaps we can fix this; i was not sure if we might want to completely drop `SCAP_KERNEL_MODULE_NAME` so i didn't touch it.